manifests/init.pp

```diff
@@ -40,11 +40,21 @@
 #  The sys_id of the user assigned to the incident as specified in the
 #  sys_user table. Note that if assignment_group is also specified, then
 #  this must correspond to a user who is a member of the assignment_group.
+# @params [Array[Enum['failed_changes', 'corrective_changes', 'intentional_changes', 'all', 'none'], 1, 3]] incident_creation_report_statuses
+#   The kinds of report attributes that can trigger an incident to be sent to
+#   Servicenow. Choose any of ['failed_changes', 'corrective_changes',
+#   'intentional_changes', 'pending_changes', 'all', 'none', 'no_changes'] You
```

Comment:
Do you think we should clarify what `no_changes` mean or could users figure that out? I lean towards some clarification since it's better than `unchanged` but still a bit implicit.

Reply:
We can add a blurb.